modules/primer-marketing-buttons/README.md

```diff
@@ -0,0 +1,91 @@
+# Primer Marketing CSS Typography
+
+[![npm version](http://img.shields.io/npm/v/primer-marketing-type.svg)](https://www.npmjs.org/package/primer-marketing-type)
+[![Build Status](https://travis-ci.org/primer/primer-css.svg?branch=master)](https://travis-ci.org/primer/primer-css)
+
+> Flash messages, or alerts, inform users of successful or pending actions. Use them sparingly. Don’t show more than one at a time.
```

Comment:
And this description should be for marketing buttons too... now going to check if this description is in marketing typography O_o

Reply:
so it _should_ be in marketing buttons? it was copied and pasted from marketing type. i just copy and pasted the message from primer-buttons instead.